internal/crypt/crypt_impl_macos.go

```diff
@@ -0,0 +1,7 @@
+// +build macos
+
+package crypt
+
+func crypt(pass, salt string) (string, error) {
+	return "", nil
```

Comment:
Why is this here? This seems like the kind of thing that can come back to haunt you in the future...

Reply:
yeah, @ondrejbudai advised me to use panic instead, which is very, very resonable :)

Reply:
And it's here to make compilation on macos possible, because glibc does not support macos so the implementation in `crypt_impl` cannot be compiled...